gammapy/detect/test_statistics.py

```diff
@@ -323,13 +323,22 @@ def compute_ts_map(counts, background, exposure, kernel, mask=None, flux=None,
     ----------
     [Stewart2009]_
     """
-    from scipy.ndimage.morphology import binary_erosion
     from time import time
     t_0 = time()
 
     assert counts.shape == background.shape
     assert counts.shape == exposure.shape
 
+    # in some maps there are pixels, which have exposure, but zero
+    # background, which doesn't make sense and causes the TS computation
+    # to fail, this is a temporary fix
+    mask_ = np.logical_and(background == 0,  exposure > 0)
```

Comment:
I meant `&` where I typed `&&`.
If you think `np.logical_and` is better than `&`, sure, OK.